cmd/terramate/cli/cloud.go

```diff
@@ -78,12 +78,41 @@ func (c *cli) checkSyncDeployment() {
 		fatal(err)
 	}
 
-	if orgs := c.cred().organizations(); len(orgs) != 1 {
+	// at this point we know user is onboarded, ie has at least 1 organization.
+	orgs := c.cred().organizations()
+
+	useOrgName := os.Getenv("TM_CLOUD_ORGANIZATION")
+	if useOrgName != "" {
+		var useOrgUUID string
+		for _, org := range orgs {
+			if org.Name == useOrgName {
+				if org.Status != "active" {
+					fatal(errors.E("You are not yet an active member of organization %s. Please accept the invitation first.", useOrgName))
+				}
+
+				useOrgUUID = org.UUID
+				break
+			}
+		}
+
+		if useOrgUUID == "" {
+			fatal(errors.E("You are not a member of organization %q or the organization does not exist. Available organizations: %s",
```

Comment:
Nitpick: Formatting an org list as a single string is going to be awkward to read and awkward to interact with, doesn't make for a great displayed message. Fine for now, not actually much of a problem, should be addressed if we make a big sweep of improving log/user messages.